packages/nextjs/app/builders/0x981ab0D817710d8FFFC5693383C00D985A3BDa38/page.tsx

```diff
@@ -0,0 +1,52 @@
+"use client";
+
+import { useEffect, useState } from "react";
+import Link from "next/link";
+import type { NextPage } from "next";
+import { useEnsAddress, useEnsAvatar } from "wagmi";
+import { BlockieAvatar } from "~~/components/scaffold-eth";
+
+const ensName = "zapaz.eth";
+
+const Zapaz: NextPage = () => {
+  const [ensAdddress, setEnsAddress] = useState<string | null>();
```

Comment:
Small typo - should be endAddress (two d's, not three)

Reply:
fixed